static/app/components/organizations/multipleProjectSelector.tsx

```diff
@@ -35,6 +36,12 @@ type Props = WithRouterProps & {
   showProjectSettingsLink?: boolean;
   lockedMessageSubject?: React.ReactNode;
   footerMessage?: React.ReactNode;
+  customDropdownButton?: (config: {
```

Comment:
unfortunate that we have to add these extra props, but this will be temporary until we remove the old GSH for good and can just have the new page filter be the only look of this component